spec/support/helpers/journey_helper.rb

```diff
@@ -26,5 +26,19 @@ def stub_participant_validation_request(trn: "1234567", date_of_birth: "1980-12-
         )
         .to_return(status: 200, body: participant_validator_response(**response), headers: {})
     end
+
+    def stub_previously_funded_request(get_an_identity_id:, npq_course_identifier:, trn: "1234567")
```

Comment:
There may already be a helper in place that covers this, I added it in a recent PR, `mock_previous_funding_api_request` stubs this and lets you set a body if you want one.

Reply:
Some reason thought this wasnt available but will double check